docs/databags.md

```diff
@@ -93,7 +93,10 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `['bag']` - A string that points to an externalized Apps Hash in which all keys (except this one) are valid. 
 * `[app]` - The name of an app to manage (disk name)
```

Comment:
As I was reading this and looking at [the example below](https://github.com/cerner/cerner_splunk/pull/48/files#diff-cb44ff154fe5f501a6e91864def7eccaR4), I got to wondering about the expected structure of the tarball.  The way I have seen app tarballs (and the way our team built our own tarball) is with a top-level directory that has the app name but since you have a name here it is somehwhat ambiguous as to whether the tarball is expected to have that top-level directory layer.  If I'm reading the code correctly you do expect that layer to be there but it would be nice to have that made a bit more explicit in the doc.  Something else related would be question as to what happens if the name of that top-level directory differs from the name specified here?